meshtastic/interfaces/ble/receive_service.py

```
@@ -33,82 +34,176 @@
+            raw_is_closing = getattr(iface, "_is_connection_closing", False)
+            state_is_closing = self._normalize_bool_probe(raw_is_closing)
+        else:
+            raw_is_closing = getattr(state_manager, "is_closing", None)
+            if callable(raw_is_closing) and not _is_unconfigured_mock_callable(
+                raw_is_closing
+            ):
+                try:
+                    result = raw_is_closing()
+                    state_is_closing = result if isinstance(result, bool) else False
+                except (
+                    Exception
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)